src/Shopware/AbstractTask.php

```diff
@@ -16,10 +16,19 @@
      */
     protected function getPathToPhpExecutable()
     {
-        return
-            ($this->runtime->getConfigOption('php_executable') !== null)
-            ? $this->runtime->getConfigOption('php_executable')
-            : 'php'
-        ;
+        $phpExecutable = 'php';
+
+        $configPhpExecutable = $this->runtime->getConfigOption('php_executable');
+        $environmentPhpExecutable = $this->runtime->getEnvOption('php_executable');
+
+        if ($configPhpExecutable !== null) {
+            $phpExecutable = $configPhpExecutable;
```

Comment:
I'd probably use early return so it's more clear what is supposed to happen instead of overwriting the same variable twice.